DFE-Digital/early-years-foundation-reform
# remove other useful resources section

## Ticket and context

Ticket: [HFEYP-629](https://dfedigital.atlassian.net/browse/HFEYP-629)
## Tech review
Remove "Other useful resources" section from the bottom of articles

### Is there anything that the code reviewer should know?

### Code quality checks

## Product review

### How can someone see it in review app?
1. Click [this link to view an article](https://eyfs-cms-review-pr-542.london.cloudapps.digital/articles/test-article) in review app
2. Verify that the "Other useful resources" section has been removed from articles. Before and after screenshots are [available in the Jira task](https://dfedigital.atlassian.net/browse/HFEYP-629)
